### DISCUSSION TITLE
How to deploy minio in Linux production environment and use two domain names for nginx external and internal access proxy? And start the account and password to log in to the management interface and report 401

### DISCUSSION THREAD

**Discussion by wdjmirrors** [minio/minio#20539] [Q&A]
My startup script start-minio.sh configured:

export MINIO_ROOT_USER=root
export MINIO_ROOT_PASSWORD=xxxxxxxx
export MINIO_SERVER_URL=https://xxxxx.cn/
export MINIO_BROWSER_REDIRECT_URL=https://xxxxxx.cn/platform/files/

./minio server --address 0.0.0.0：9000 --console-address 0.0.0.0：9111 /usr/local/minio/data > /usr/local/minio/minio.log 2>&1 & &

**Comment by marktheunissen** [comment]
This is a very broad question that is covered in our getting started documentation: https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html

Don't use a bash script like that in a production environment. Use our documented systemd config as above.
